_extensions/posit-docs/_extension.yml

```diff
@@ -21,11 +20,10 @@ contributes:
              href: "https://docs.posit.co"
            - text: "Posit Support"
              href: "https://support.posit.co/hc/en-us/"
-      sidebar:
-        style: "floating"
-        collapse-level: 1
-        search: true
-        pinned: false
+      search:
+        copy-button: true
+        show-item-context: true
+    format: posit-docs-html
```

Comment:
Why was this added? We ran ito issues with having format and html included before and had to remove it.

Reply:
Sets a default format for users of this custom project type. Problems can happen when folks try to provide overrides to both `format: html` and `format: posit-docs-html`; they can use only one.